examples/apps/generic_client/generic_client.py

```diff
@@ -116,14 +115,203 @@ def ParseCommandLine(args):
         json.dumps(config)
     except pconfig.ConfigurationException as e:
         logger.error(str(e))
+        config = None
+
+    return config
+
+
+def _retrieve_uri_from_registry_list(config):
+    # Retrieve Http JSON RPC listener uri from registry
+    logger.info("\n Retrieve Http JSON RPC listener uri from registry \n")
+    # Get block chain type
+    blockchain_type = config['blockchain']['type']
+    if blockchain_type == "Ethereum":
+            worker_registry_list = EthereumWorkerRegistryListClientImpl(
+                config)
+    else:
+        worker_registry_list = None
+        logger.error("\n Worker registry list is currently supported only for "
+                     "ethereum block chain \n")
+        return None
+
+    # Lookup returns tuple, first element is number of registries and
+    # second is element is lookup tag and
+    # third is list of organization ids.
+    registry_count, lookup_tag, registry_list = \
+        worker_registry_list.registry_lookup()
+    logger.info("\n Registry lookup response: registry count: {} "
+                "lookup tag: {} registry list: {}\n".format(
+                    registry_count, lookup_tag, registry_list))
+    if (registry_count == 0):
+        logger.error("No registries found")
+        return None
+    # Retrieve the fist registry details.
+    registry_retrieve_result = worker_registry_list.registry_retrieve(
+        registry_list[0])
+    logger.info("\n Registry retrieve response: {}\n".format(
+        registry_retrieve_result
+    ))
+
+    return registry_retrieve_result[0]
+
+
+def _lookup_first_worker(worker_registry, jrpc_req_id):
+    # Get first worker id from worker registry
+    worker_id = None
+    worker_lookup_result = worker_registry.worker_lookup(
+        worker_type=WorkerType.TEE_SGX, id=jrpc_req_id
+    )
+    logger.info("\n Worker lookup response: {}\n".format(
+        json.dumps(worker_lookup_result, indent=4)
+    ))
+    if "result" in worker_lookup_result and \
+            "ids" in worker_lookup_result["result"].keys():
+        if worker_lookup_result["result"]["totalCount"] != 0:
+            worker_id = worker_lookup_result["result"]["ids"][0]
+        else:
+            logger.error("ERROR: No workers found")
+            worker_id = None
+    else:
+        logger.error("ERROR: Failed to lookup worker")
+        worker_id = None
+
+    return worker_id
+
+
+def _create_work_order(worker_id, workload_id, in_data, worker_encrypt_key,
```

Comment:
It is creating work order request not submitting request. change it to _create_work_order =>_create_work_order_request

Reply:
or should we rename it to _create_work_order_params ? This function is returning WorkOrderParams object